cEP-0001.md

```diff
@@ -1,25 +1,20 @@
-Decision Making in coala
-========================
+# Decision Making in coala
 
-|Metadata|                                                                    |
-|--------|--------------------------------------------------------------------|
-|cEP     |0001                                                                |
-|Version |2.0                                                                 |
-|Title   |Decision Making in coala                                            |
-|Authors |Maximilian Scholz <max.scholz@mailbox.org>, Lasse Schuirmann <lasse@gitmate.io>, Mischa Krüger <makman@alice.de>|
-|Status  |Active                                                              |
-|Type    |Process                                                             |
+| Metadata |                                                                                                                                       |
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| cEP      | 0001                                                                                                                                  |
+| Version  | 2.0                                                                                                                                   |
+| Title    | Decision Making in coala                                                                                                              |
+| Authors  | Maximilian Scholz <mailto:max.scholz@mailbox.org>, Lasse Schuirmann <mailto:lasse@gitmate.io>, Mischa Krüger <mailto:makman@alice.de> |
```

Comment:
this is going to be annoying when another author is added.
markdown tables suck

Reply:
Make it 1 author per line?

Reply:
then we get into variations of markdown.
not worth it atm.
IMO we propose a change to cEP 0 to replace .md with .asciidoc or rst. 

https://github.com/coala/cEPs/issues/129